README.md

```diff
@@ -1,5 +1,31 @@
 # Introduction 
-Project Coimbra
+Project Coimbra 
+
+# Setup for development
+1. Clone https://github.com/melanchall/drywetmidi/  , and put the repo in this structure 
+<pre>
+Coimbra repo  
+ |-.gitignore  
+ |-README.md  
+ |-...  
+ |-ProjectCoimbra.UWP  
+   |-Project.Coimbra  
+   |-Project.Communication
+   |-...  
+   |-<b>drywetmidi</b>
+	 |-.gitignore  
+	 |-README.md  
+	 |-...
+	 |-<b>DryWetMidi</b>
+	   |-<b>Melanchall.DryWetMidi.csproj</b>
+</pre>
+2. Launch solution ProjectCoimbra.UWP/Project.Coimbra.sln
+3. Create your own certificate for development by  
+3.1. In Visual Studio, Solution Explorer, open Project.Coimbra (Universal Windows) / Package.appxmanifest   
+3.2. Go to <b>Packaging</b> tab, click <b>Choose Certificate...</b>, then click <b>Create...</b> to create your own certificate for development  

```

Comment:
For bold in Markdown, use ** rather than HTML tags. In general, you should avoid HTML tags in Markdown unless there is no alternative